app/assets/javascripts/views/data_portal/FiltersIndicatorsModal.js

```diff
@@ -0,0 +1,101 @@
+(function (App) {
+
+  App.View.FilterIndicatorsModal = App.Component.Modal.extend({
+
+    defaults: {
+      // See App.Component.Modal for details about this option
+      title: 'Select indicators',
+      // See App.Component.Modal for details about this option
+      showTitle: true,
+      // See App.Component.Modal for details about this option
+      footer: '<button class="c-button -white -outline -padding js-cancel">Cancel</button><button class="c-button -white -padding js-done">Done</button>',
+      // settings for default tab to show
+      tab: {
+        name: 'apply-filters',
+        index: 1
+      },
+      // Callback executed when the user presses the "Done" button
+      // The callback gets passed the name of the selected chart
+      continueCallback: function () {}
+    },
+
+    events: function () {
+      return _.extend({}, App.Component.Modal.prototype.events, {
+        'click .js-cancel': 'onCloseModal',
+        'click .js-done': '_onClickDone'
+      });
+    },
+
+    contentTemplate: JST['templates/data_portal/filters'],
+
+    initialize: function (options) {
+      this.constructor.__super__.initialize.call(this, options);
+      this.indicators = options.indicators;
+      this.filters = options.filters;
+
+      this._setVars();
+      this.render();
+    },
+
+    _setVars: function () {
+      this.constructor.__super__._setVars.apply(this);
+
+      this.pages = [
+        {
+          id: 'select-indicators',
+          view: App.View.SelectIndicatorsView
+        },
+        {
+          id: 'apply-filters',
+          view: App.View.ApplyFiltersView
+        }
+      ];
+    },
+
+    _setEventListeners: function () {
+      this.constructor.__super__._setEventListeners.apply(this);
+
+      Backbone.Events.on('tab:selected', function (tab) {
```

Comment:
I wouldn't use `Backbone.Events` here because if we have another tab in the page somewhere, this will get triggered. Why not using `this.listenTo(FilterView, 'tab:selected', this._onTabSelected);`?